packages/esm-ward-app/src/ward-view-header/ward-metric.component.tsx

```diff
@@ -1,20 +1,27 @@
 import React from 'react';
 import styles from './ward-metric.scss';
 import { SkeletonPlaceholder } from '@carbon/react';
+import {useTranslation } from 'react-i18next';
+import classNames from 'classnames';
 
 interface WardMetricProps {
   metricName: string;
   metricValue: string;
   isLoading: boolean;
 }
 const WardMetric: React.FC<WardMetricProps> = ({ metricName, metricValue, isLoading }) => {
+  const { i18n } = useTranslation();
```

Comment:
Usually, this is done as a interpolated translation string, like this:

https://github.com/openmrs/openmrs-esm-patient-management/blob/6822458dd561589be4ed3b808b04c838aa508b5d/packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-obs.resource.ts#L51-L54

Reply:
Hi @chibongho , removed i18n here , seems the direction automatically handeled in the document. Thanks